stdlib/_ast.pyi

```diff
@@ -375,3 +375,37 @@ class alias(AST):
 class withitem(AST):
     context_expr: expr
     optional_vars: Optional[expr]
+
+if sys.version_info >= (3, 10):
+    class Match(stmt):
+        subject: expr
+        cases: typing.List[match_case]
+    class pattern(AST): ...
+    # Without the alias, Pyright complains variables named pattern are recursively defined
+    _pattern = pattern
+    class match_case(AST):
+        pattern: _pattern
+        guard: Optional[expr]
+        body: typing.List[stmt]
+    class MatchValue(pattern):
+        value: expr
+    class MatchSingleton(pattern):
+        value: expr
```

Comment:
According to the docs value is of type `constant` (similar to `ast.Constant`).

Reply:
Thanks, I changed to Optional[bool] because https://docs.python.org/3.10/library/ast.html#ast.MatchSingleton